### PR TITLE
Update configure-language-version.md

### PR DESCRIPTION
**Pull request by zzfima**
C# 11 add info

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/language-reference/configure-language-version.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ms.date: 02/25/2022
...
 
-[C# 10](../whats-new/csharp-10.md) is supported only on .NET 6 and newer versions. [C# 9](../whats-new/csharp-9.md) is supported only on .NET 5 and newer versions. [C# 8.0](../whats-new/csharp-8.md) is supported only on .NET Core 3.x and newer versions.
+[C# 11](../whats-new/csharp-11.md) is supported only on .NET 6.0.200 and newer versions. [C# 10](../whats-new/csharp-10.md) is supported only on .NET 6 and newer versions. [C# 9](../whats-new/csharp-9.md) is supported only on .NET 5 and newer versions. [C# 8.0](../whats-new/csharp-8.md) is supported only on .NET Core 3.x and newer versions.
```
Comment (Youssef1313) [inline docs/csharp/language-reference/configure-language-version.md:12]: I'm not sure this is correct. C# 11 should be *officially* supported on .NET 7 and later.

File: docs/csharp/language-reference/configure-language-version.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ ms.date: 02/25/2022
 
 The latest C# compiler determines a default language version based on your project's target framework or frameworks. Visual Studio doesn't provide a UI to change the value, but you can change it by editing the *csproj* file. The choice of default ensures that you use the latest language version compatible with your target framework. You benefit from access to the latest language features compatible with your project's target. This default choice also ensures you don't use a language that requires types or runtime behavior not available in your target framework. Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors.
```
Comment (BillWagner) [inline docs/csharp/language-reference/configure-language-version.md:10]: Adding a sentence here to clarify preview usage:

```suggestion
The latest C# compiler determines a default language version based on your project's target framework or frameworks. Visual Studio doesn't provide a UI to change the value, but you can change it by editing the *csproj* file. The choice of default ensures that you use the latest language version compatible with your target framework. You benefit from access to the latest language features compatible with your project's target. This default choice also ensures you don't use a language that requires types or runtime behavior not available in your target framework. Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors. In some cases, preview features are enabled to gather feedback on those features. Those scenarios aren't officially supported.
```

File: docs/csharp/language-reference/configure-language-version.md [hunk 3]
```diff
@@ -19,7 +19,8 @@ The compiler determines a default based on these rules:
 
 | Target framework | version | C# language version default |
 |------------------|---------|-----------------------------|
-| .NET             | 6.x     | C# 10                     |
+| .NET             | 6.0.200 | C# 11                       |
```
Comment (BillWagner) [inline docs/csharp/language-reference/configure-language-version.md:22]: Same change here:

```suggestion
| .NET             | 7.x | C# 11                       |
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks @zzfima 

As @Youssef1313 points out, using C# 11 with the .NET 6 SDK is enabled, but not supported. I made some suggestions to clarify that.

Once you accept those suggestions, this is ready to merge.

**Review by zzfima — COMMENTED** [review]
ok

**Review by Youssef1313 — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks again @zzfima 

I've approved this, and it will merge once the build completes.

**Pull request merged**